doc/api/errors.md

```diff
@@ -881,6 +881,12 @@ provided.
 Encoding provided to `TextDecoder()` API was not one of the
 [WHATWG Supported Encodings][].
 
+<a id="ERR_EXECUTION_ENVIRONMENT_NOT_AVAILABLE"></a>
+### `ERR_EXECUTION_ENVIRONMENT_NOT_AVAILABLE`
+
+JS execution context not associated with Node.js environment (this happens
+mostly under out of resources / internal fault situations).
```

Comment:
This is not accurate.

```suggestion
The JS execution context is not associated with a Node.js environment.
This may occur when Node.js is used as an embedded library and some hooks
for the JS engine are not set up properly.
```

Reply:
@addaleax, thanks. Replaced it with your suggestion.